config/desktop.json

```diff
@@ -41,7 +37,6 @@
 		"google-my-business": false,
 		"gutenberg": false,
 		"gutenberg/opt-in": false,
-		"gutenberg/block/jetpack-preset": false,
```

Comment:
Gutenberg isn't even running in Desktop, right? so should we remove this flag here?

Reply:
I generally like keeping flags around and explicitly setting vs depending on default config behavior.

Reply:
Yeah, so I think it makes sense to remove the `jetpack-preset` flag altogether and treat that feature as a subset of the `gutenberg` feature...

Reply:
> I generally like keeping flags around and explicitly setting vs depending on default config behavior.

Hmm, I thought Calypso has a bunch of dead code branches though because of feature flags that were never removed.  (I've been certainly guilty of that.)